src/portal/src/app/project/repository/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss

```diff
@@ -1,5 +1,5 @@
-$thirty-pixel: 30px;
-$twenty-two-pixel: 22px;
+$thirty-pixel: 26px;
+$twenty-two-pixel: 18px;
 
```

Comment:
![image](https://user-images.githubusercontent.com/29826789/84231367-924c5280-ab20-11ea-9864-4397f5311cbe.png)
![image](https://user-images.githubusercontent.com/29826789/84231425-aee88a80-ab20-11ea-81ea-2bb1fd0d8ba0.png)
The background color is very strange, it is recommended to find pm to confirm that this is the color he wants

Reply:
Fixed and comfirmed with PM